meerkat_frontend/src/js/technical/misc.js

```diff
@@ -1,3 +1,35 @@
+
+//This Code will handle the spinner all over the site ...
+$body = $("body");
+var throb;
+$(document).on({
+  ajaxStart: function() {
+      // here i can control the spinner properties ...
+      $('#divSpinner').empty();
```

Comment:
Should this use the eclipse spinner gif as a fall back?  I think I saw something about a fall back option?  We should probably either use the eclipse spinner gif or remove it from the pull request.